modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/CopyDataDefinitionMVCActionCommand.java

```diff
@@ -75,26 +74,23 @@ protected void doTransactionalCommand(
 		Map<Locale, String> nameMap = _localization.getLocalizationMap(
 			actionRequest, "name");
 
-		DataDefinitionResource.Builder dataDefinitionResourceBuilder =
+		DataDefinitionResource.Builder dataDefinitionResourcedBuilder =
 			_dataDefinitionResourceFactory.create();
 
 		DataDefinitionResource dataDefinitionResource =
-			dataDefinitionResourceBuilder.user(
+			dataDefinitionResourcedBuilder.user(
```

Comment:
@devpedrotavares please rename it back to `dataDefinitionResourceBuilder`

Reply:
Done!